### PR TITLE
Fix line breaks in announcement email content

### PR DESCRIPTION
**Pull request by Vignaraj-pai**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

Fixes # (issue)

## Dependencies

List any dependencies that are required for this change.

## Type of Change

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration. (for bug fix / breaking change)
_Put an `x` in the boxes that apply_

- [ ] Test A
- [ ] Test B

---

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: corpus/impulse/views.py [hunk 1]
```diff
@@ -405,15 +405,25 @@ def announcements_management(request):
...
                     )
+                    users = users.exclude(is_staff=True)
+                    users = users.exclude(is_superuser=True)
+                    
+                    email_ids = list(users.values_list("email", flat=True))
+                    
                     
             if email_ids is not None:
+                # THIS REPLACES \n WITH <br> IN THE EMAIL CONTENT
+                announcement.content = announcement.content.replace("\n", "<br>")
```
Comment (anirudhprabhakaran3) [inline corpus/impulse/views.py:426]: Don't use this, use `linebreaks` in the template.

https://stackoverflow.com/a/10270934

File: corpus/templates/emails/impulse/announcement.html [hunk 2]
```diff
@@ -6,7 +6,9 @@
 
 {% block content %}
     <h1>Announcement - Impulse</h1>
-    <p>{{ announcement.content }}</p>
+    <div style="font-size: 1.2em;">
+        {{ announcement.content | safe }}
```
Comment (anirudhprabhakaran3) [inline corpus/templates/emails/impulse/announcement.html:10]: As mentioned above

Comment (anirudhprabhakaran3) [reply corpus/templates/emails/impulse/announcement.html:10]: You should also update this in the HTML page that you are displaying to participants. 

Also, could you make the same change in the corresponding places in Embedathon, i.e. in the mail template and the participant dashboard?

**Review by anirudhprabhakaran3 — CHANGES_REQUESTED** [review]

File: corpus/impulse/views.py (hunk 1 above)
Comment (Vignaraj-pai) [reply corpus/impulse/views.py]: Done.

File: corpus/templates/emails/impulse/announcement.html (hunk 2 above)
Comment (Vignaraj-pai) [reply corpus/templates/emails/impulse/announcement.html]: Yep, replaced in all templates.

**Review by anirudhprabhakaran3 — APPROVED** [review]

**Pull request merged**